### ISSUE TITLE
filter by rows and columns number

### ISSUE THREAD

**Issue by aborruso** [johnkerl/miller#195]
Hi,
is there a way to filter all the cells of rows 3 and 7 and first and third columns ?

Something like (supposing that NC is Column Number)

    mlr --csv filter  '(NR == 3 || NR == 7) && (NC == 1 || NC == 3)' input.csv

Thank you

**Comment by johnkerl** [comment]
I think you want `mlr cut` for the column select and `filter` for the row-select

**Comment by johnkerl** [comment]
yes just unix `cut`. or, `mlr --nidx --fs comma cut -f 1,3` but NIDX format won't handle double-quoting as intelligently as full CSV format would.

**Comment by aborruso** [comment]
@johnkerl only to be sure to have understood you.

Than to filter cells of rows 3 and 7 and first and third columns, the right command is:

    mlr --csv --fs comma cut -f 1,3 then filter '(FNR == 3 ) || (FNR == 7 )' table.csv

Am I right?

**Comment by aborruso** [comment]
no @johnkerl sorry, it can not be done, I misunderstood
